docs/src/pages/sveltekit.mdx

```diff
@@ -0,0 +1,257 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Getting Started with SvelteKit
+
+Make sure you
+[install the packages and get your env variables first](/getting-started)
+
+## Package Setup
+
+<Steps>
+
+### Install the packages
```

Comment:
Increasingly feel like I need to make a shared component for this setup that takes the `additionalPackages` as props. Not for this PR, just a note for myself for the future